backend/src/main/java/pl/edu/agh/awiteks_backend/configs/SwaggerConfig.java

```diff
@@ -0,0 +1,23 @@
+package pl.edu.agh.awiteks_backend.configs;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.security.SecurityScheme;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class SwaggerConfig {
```

Comment:
Apropo swaggera, to w pliku `AuthConstansConfig` pasuje zmienić tego beana na:
```java
    @Bean(name = "jwtExcludedRoutes")
    public List<String> configureJwtExcludedRoutes() {
        return List.of(
                "/auth/.*", // auth endpoints
                "/v3/.*",    // swagger endpoints
                "/swagger-ui/.*"        // swagger UI
            );
    }
```
Bo chciałbym korzystać z tego UI